django_ses_plus/models.py

```diff
@@ -0,0 +1,60 @@
+import uuid
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.db import models
+from django.template.loader import render_to_string
+from django.utils import translation
+from django.utils.translation import ugettext_lazy as _
+
+from .utils import sent_email_upload_path
+
+
+class SentEmail(models.Model):
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+
+    recipient = models.ForeignKey(settings.AUTH_USER_MODEL, related_name="sent_emails", on_delete=models.SET_NULL, null=True)
+    to_email = models.EmailField()
+    from_email = models.EmailField()
+    subject = models.TextField()
+    html = models.FileField(upload_to=sent_email_upload_path)
+
+    creation_datetime = models.DateTimeField(auto_now_add=True)
+    update_datetime = models.DateTimeField(auto_now=True)
```

Comment:
I didn't quite get why we need `update_datetime` here. Can a `SentEmail` be updated later on?

Reply:
Right now it's not updated. But once we have the delivery status fields it will be updated.